src/components/BookPlace.tsx

```diff
@@ -36,8 +38,35 @@ const BookPlace: React.FC<BookPlaceProps> = ({ onClose, placeId, placeBookings,
   }
 
   // Getting info from inputs
-  const [visitorName, setVisitorName] = useState<string>('')
-  const onNameChange = (event: React.ChangeEvent<HTMLInputElement>) => setVisitorName(event.target.value)
+  const { members } = useMembersCollection()
+
+  const [visitorName, setVisitorName] = useState('')
```

Comment:
The state can be an object
```suggestion
  const [visitor, setVisitor] = useState(null)
```

```
setVisitor({ name, id })
```